contents/blog/posthog-vs-optimizely.md

```diff
@@ -0,0 +1,616 @@
+ </tr>
+ </tbody>
+</table>
+</div>
+
+In Optimizely, many of the feature experimentation features, like multi-armed bandits and custom segmentation, are only available are the higher tier plans. These are all available for free in PostHog.
+
+## Pricing
+
+PostHog and Optimizely differ significantly in their approach to pricing. Optimizely is sales-driven and completely opaque. PostHog is self-serve and transparent.
+
+<div className="overflow-x-auto -mx-5 px-5 w-screen md:w-auto">
+<table className="w-full mt-4">
+ <thead>
+ <tr>
+ <td className="w-3/12"></td>
+ <td class="text-center"><strong>PostHog</strong></td>
+ <td class="text-center"><strong>Optimizely</strong></td>
+ </tr>
+ </thead>
+ <tbody>
+ <tr>
+ <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Transparent</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">See pricing without talking to sales</p></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td>
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ </tr>
+ <tr>
+ <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Usage-based</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Pay based on how much you use</p></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ </tr>
+ <tr>
+ <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Free tier</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Use for free forever</p></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ </tr>
+ <tr>
+ <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Per-product pricing</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Different pricing</p></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-green text-lg">✔</span></td>
+ </tr>
+ </tbody>
+</table>
+</div>
+
+Both PostHog and Optimizely do per-product pricing. This means each product has its own pricing. Each PostHog product has free forever pricing as well as a generous free tier on paid plans.
+
+Products with pricing for each include:
+
+- Optimizely: content marketing platform, content management system, web experimentation, feature experimentation, configured commerce, customized commerce.
+
+- PostHog: product analytics, session replay, feature flags, A/B testing, surveys.
+
+For many Optimizely products, there are also paid add-ons like advanced personalization, the data platform, and their Salesforce integration. PostHog only has free and paid tiers for each product with a single addition enterprise add-on for SSO, dedicated support, and advanced permissions.
+
+![Flag pricing](../images/blog/posthog-vs-optimizely/pricing.png)
```

Comment:
Couple of comments:

1. This gets into the weeds a bit too much. I don't feel like I need to know this much
2. The pricing page screenshot is a bit weird because it looks like one of our comparison tables. Best to remove, I think.